packages/core/src/button/button.element.ts

```diff
@@ -122,7 +130,6 @@ export class CdsButton extends CdsBaseButton {
   static styles = [baseStyles, styles];
 
   private async disableButton() {
```

Comment:
If we removed the `await` we probably don't need the `async` declaration either...